lib/features/profiles/cubit/profiles_cubit.dart

```diff
@@ -18,34 +18,87 @@ class ProfilesCubit extends HydratedCubit<ProfilesState> {
 
   final ProfilesRepository _profilesRepositoy;
 
-  Future<void> fetchProfiles(String parentGuid) async {
-    final activeProfileBalance = state.activeProfile.wallet.balance;
+  Future<void> fetchProfilesOld(String parentGuid) async {
```

Comment:
This method can be removed I think? :) (comment about the countdown is also below)

Reply:
Yes I replaced it in the editing cubit as well, its no longer used.